xcodeproj/internal/xcodeproj_runner.bzl

```diff
@@ -30,6 +30,8 @@ build:{config}_swiftuipreviews --config=rules_xcodeproj_swiftuipreviews
 build:{config}_swiftuipreviews --config={config}
 build:{config}_tsan --config=rules_xcodeproj_tsan
 build:{config}_tsan --config={config}
+build:{config}_asan --config=rules_xcodeproj_asan
```

Comment:
Can you put this above the thread sanitizer?

Reply:
Done, thanks!